cypress/integration/mocked/okta_register.3.cy.ts

```diff
@@ -125,7 +129,8 @@ describe('Okta Register flow', () => {
       cy.contains('user@example.com');
       cy.contains('Continue')
         .should('have.attr', 'href')
-        .and('include', 'https://m.code.dev-theguardian.com');
+        .and('include', '/signin/refresh')
+        .and('include', 'returnUrl=https%3A%2F%2Fm.code.dev-theguardian.com');
```

Comment:
How come the `returnUrl` on line 64 has a slash at the end but the one here doesn't? Weird.

Reply:
I think the test was written with a slash in one test, and without a slash in the other. It doesn't matter too much